tests/providers/openai/test_openai.py

```diff
@@ -0,0 +1,39 @@
+from typing import Tuple
+
+import os
+import pytest
+
+from exchange import Text
+from exchange.message import Message
+from exchange.providers.base import Usage
+from exchange.providers.openai import OpenAiProvider
+from .conftest import OPENAI_MODEL, OPENAI_API_KEY
+
+
+@pytest.mark.vcr()
+def test_openai_completion(monkeypatch):
```

Comment:
so you can imagine we can add another test here for reasoning model, and then we can see the output, which would have reasoning tokens in it. like `test_openai_completion_reasoning`